test/jdk/javax/swing/JMenuBar/TestMenuMnemonic.java

```diff
@@ -105,6 +88,27 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    private static void verifyMnemonicsState() {
+        MenuSelectionManager msm =
+                MenuSelectionManager.defaultManager();
+        MenuElement[] selectedPath = msm.getSelectedPath();
+        if (WindowsLookAndFeel.isMnemonicHidden()) {
+            mnemonicHideCount.getAndIncrement();
+            // check if selection is cleared when mnemonics are hidden
```

Comment:
We capitalise ‘C‘ since it starts a sentence:
```suggestion
            // Check if selection is cleared when mnemonics are hidden
```

See [a comment on the subject](https://github.com/openjdk/jdk/pull/18067#discussion_r1511570584).